tests/inheritance.test.ts

```diff
@@ -0,0 +1,164 @@
+import { assertPrint, assertFail, assertTCFail, assertTC } from "./asserts.test";
+import { NUM, BOOL, NONE, CLASS } from "./helpers.test"
+
+describe("Inheritance test", () => {
+    assertPrint("Field Access", `class A(object):
+    a : int = 1
+class B(A):
+    b : int = 2
+x : B = None
+x = B()
+print(x.a)
+print(x.b)`, ["1","2"]);
+
+   assertPrint("Method Call",`class A(object):
+   a : int = 1
+   def get(self: A)->int:
+       return self.a
+class B(A):
+   c : int = 3
+   b : int = 2
+   
+x : B = None
+x = B()
+print(x.get())`,[`1`]);
+
+   assertTCFail("None Super Class",`class Test(None):
+x : int = 0`);
+
+   assertTCFail("Parent-child same field",`
```

Comment:
Does this have to be a TC failure? CPython has semantics for this, but ChocoPy considers this an error. Any reasoning behind your choice?

Reply:
It indeed does not have to be a TC failure.
[This StackOverflow post](https://stackoverflow.com/questions/12409714/python-class-members) explains some of the trickiness with CPython.
The baseline compiler we started with does not support the notion of "class variables," only "instance variables." To be honest, we didn't ever consider adding support for class variables.
I suppose we could change this, but I'm wary of the amount of effort needed to implement class variable assignment and lookup. Maybe something for next steps in addition to/instead of multiple inheritance?